pkg/utils/utils.go

```diff
@@ -172,3 +172,21 @@ func GetNumVpnConnections(cloud1, cloud2 string) int {
 	}
 	return 1
 }
+
+// DoCIDROverlap returns false if cidr blocks don't share a single ip,
+// i.e. they don't overlap.
+func DoesCIDROverlap(cidr1, cidr2 string) (bool, error) {
```

Comment:
I'll delete this function before merging. Created another PR that moves a similar function from pkg/ibm to pkg/utils so it's available to all clouds